src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/Selection.js

```diff
@@ -166,6 +234,14 @@ export default class Selection extends React.Component<Props> {
             value,
         } = this.props;
 
+        if (!Array.isArray(requestParameters)) {
+            throw new Error('The "request_parameters" schemaOption must be an array!');
+        }
+
+        if (!Array.isArray(resourceStorePropertiesToRequest)) {
```

Comment:
Wouldn't it make sense to put this check into the `buildRequestOptions` function instead of doing it twice in two different places?